modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -584,6 +584,10 @@ impl WalletClientModule {
     /// replace by fee (RBF).
     /// This can prevent transactions from getting stuck
     /// in the mempool
+    #[deprecated(
```

Comment:
Deprecating this method has the advantage of allowing us to test:
- Clients will receive an error when attempting to RBF an existing withdrawal, but the original withdrawal still succeeds
- Clients that need to enable the environment variable `FM_UNSAFE_ENABLE_RBF_WITHDRAWAL` are able to successfully process RBF withdrawals